internal/backend/remote-state/pg/backend.go

```diff
@@ -96,7 +107,42 @@ func (b *Backend) configure(ctx context.Context) error {
 	data := b.configData
 
 	b.connStr = data.Get("conn_str").(string)
-	b.schemaName = pq.QuoteIdentifier(data.Get("schema_name").(string))
+
+	schemaName, ok := data.Get("schema_name").(string)
+	if !ok {
```

Comment:
I believe it should be already set at this time. The priority list for getting that kind of values is the following:

* configuration
* env variable
* static default value

I think it is safe to leave this code as is (without type conversion checks).

Reply:
Fixed